apps/aefate/test/aefate_engine_test.erl

```diff
@@ -700,3 +700,53 @@ contracts() ->
        }.
 
 
+%% Encode is a convinience function for testing, encoding an Erlang term
```

Comment:
Can you please put this in a test util file, so it is available for other modules as well?

Reply:
done and rebased